financialaid/constants.py

```diff
@@ -52,11 +52,13 @@ class FinancialAidStatus:
         SKIPPED: "Skipped",
     }
 
-
-CURRENCY_EXCHANGE_RATE_API_REQUEST_URL = "{url}latest.json?app_id={app_id}".format(
-    url=settings.OPEN_EXCHANGE_RATES_URL,
-    app_id=quote_plus(settings.OPEN_EXCHANGE_RATES_APP_ID)
-)
+if settings.OPEN_EXCHANGE_RATES_URL and settings.OPEN_EXCHANGE_RATES_APP_ID:
```

Comment:
looking at [this line](https://github.com/mitodl/micromasters/pull/2557/files#diff-705fc88e88815b293b37d0c2adf3cccaR16), it seems like `https://openexchangerates.org/` should always be our `OPEN_EXCHANGE_RATES_URL`, or at the very least the default. `OPEN_EXCHANGE_RATES_APP_ID` probably wouldn't be relevant for any other exchange rate API apart from `openexchangerates.org`. can you set that as the `OPEN_EXCHANGE_RATES_URL` default?